README.md

```diff
@@ -100,15 +100,27 @@ $ nsecure open
 # If you want to define a specific port use the --port option.
 $ nsecure open --port 8080
 ```
----
 
-##### Available options
+### Command Documentation
+
+For complete details on each command, refer to the following documents:
+
 
-| name | shortcut | default value | description |
-| --- | --- | --- | --- |
-| --port | -p | 0 | Define the running port, can also be define through the environment variable `PORT` |
+- [Commande `cwd`](./docs/cli/cwd.md)
+- [Commande `hydrate-db`](./docs/cli/hydrate-db.md)
+- [Commande `from`](./docs/cli/from.md)
+- [Commande `auto`](./docs/cli/auto.md)
+- [Commande `open`](./docs/cli/open.md)
+- [Commande `verify`](./docs/cli/verify.md)
+- [Commande `summary`](./docs/cli/summary.md)
+- [Commande `scorecard`](./docs/cli/scorecard.md)
+- [Commande `report`](./docs/cli/report.md)
+- [Commande `lang`](./docs/cli/lang.md)
+- [Commande `config create`](./docs/cli/config-create.md)
+- [Commande `config`](./docs/cli/config.md)
 
-The `auto` command can be used to chain `cwd/from` and `open` commands automatically.
+
+Each link redirects you to the complete documentation of the command, with additional details, options, and usage examples.
 
 ```bash
 $ nsecure auto jest
```

Comment:
Please delete everything after (and before Private registry / Verdaccio section).